src/hooks/tbtc/useFetchRedemptionDetails.ts

```diff
@@ -17,11 +19,13 @@ interface RedemptionDetails {
   btcAddress?: string
 }
 
+type FetchRedemptionDetailsParamType = string | null | undefined
+
 export const useFetchRedemptionDetails = (
-  redemptionRequestedTxHash: string,
-  walletPublicKeyHash: string,
-  redeemerOutputScript: string,
-  redeemer: string
+  redemptionRequestedTxHash: FetchRedemptionDetailsParamType,
+  walletPublicKeyHash: FetchRedemptionDetailsParamType,
+  redeemerOutputScript: FetchRedemptionDetailsParamType,
+  redeemer: FetchRedemptionDetailsParamType
```

Comment:
Can't we just keep the types as string and make all the propeties optional?

```suggestion
  redemptionRequestedTxHash?: string,
  walletPublicKeyHash?: string,
  redeemerOutputScript?: string,
  redeemer?: string
```

Reply:
I decided to add `null` and create a custom type because `searchParams.get` from [here](https://github.com/threshold-network/token-dashboard/pull/566/files#diff-c0362ebe68a4f572dd5a124a7d56f3abc9086c667f2513883bcbe783181596beR74) returns `string | null` but optional param is meaning "type X or **undefined**" so I got a ts error w/o `null`.